docs/requirements.txt

```diff
@@ -0,0 +1,18 @@
+bokeh==2.4.3
+Click
+joblib
+matplotlib
+matplotlib-venn
+numpy
+pandas
+plotly
+psutil
+pyopenms
+scipy
+streamlit>=1.30.0
+streamlit-javascript
+tqdm
+upsetplot
+nbsphinx
+sphinx-copybutton
+sphinx-rtd-theme
```

Comment:
Can you reorder these based on alphanetical order, unless there is an order based on which needs to be installed first.